src/pages/index.js

```diff
@@ -68,3 +70,15 @@ export default function Home() {
     </div>
   );
 }
+
+//Initializing mongo connection
+export async function getServerSideProps() {
+  await connectMongo();
+  const result = await Test.find({})
+  const test = result.map((doc) => {
+    const pet = doc.toObject()
```

Comment:
also what is pet?

Reply:
That was a mistake, it's a legacy code from previous version I used to test db status